bin/docker-image-tool.sh

```diff
@@ -240,6 +247,7 @@ do
  X) CROSS_BUILD=1;;
  u) KYUUBI_UID=${OPTARG};;
  s) SPARK_HOME=${OPTARG};;
+ d) SPARK_HOME_IN_DOCKER=${OPTARG};;
```

Comment:
how about `S` ? `d` looks strange. And please update this option in `usage`

Reply:
fixed